--- /root/.local/venvs/pipenv/lib/python3.5/site-packages/pipenv/vendor/delegator.py    2017-03-31 09:20:36.734171152 +0000
+++ pipenv+vendor+delegator.py  2017-03-31 09:18:15.598907196 +0000
@@ -8,12 +8,14 @@
 os.environ['PYTHONUNBUFFERED'] = '1'
 
 class Command(object):
-    def __init__(self, cmd):
+    def __init__(self, cmd, stdout, stderr):
         super(Command, self).__init__()
         self.cmd = cmd
         self.subprocess = None
         self.blocking = None
         self.was_run = False
+        self.stdout = stdout
+        self.stderr = stderr
         self.__out = None
 
     def __repr__(self):
@@ -28,8 +30,8 @@
         return {
             'env': os.environ.copy(),
             'stdin': subprocess.PIPE,
-            'stdout': subprocess.PIPE,
-            'stderr': subprocess.PIPE,
+            'stdout': self.stdout,
+            'stderr': self.stderr,
             'shell': True,
             'universal_newlines': True,
             'bufsize': 0,
@@ -210,8 +212,8 @@
     return c
 
 
-def run(command, block=True):
-    c = Command(command)
+def run(command, block=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE):
+    c = Command(command, stdout, stderr)
     c.run(block=block)
 
     if block:

